### ISSUE TITLE
namespace on the initial page load is incorrect

### ISSUE THREAD

**Issue by ahmadHuss** [isaachinman/next-i18next#894]
Hey, @isaachinman Thanks for this great library. We are using this library on our website pages to handle internalization in different languages. But there is an issue that is happening on the website.

## Describe the bug
1. When we load a page or pages with different locales instead of English sometimes the namespace for that locale is not correctly loaded. From the SEO perspective crawler treating the page content as `Duplicate page without canonical`. But the content is not similar. We can ignore SEO and focus on the main issue i.e. sometimes the namespace for that locale is not correctly loaded.
2. But, after the page load when we refresh the page it will load the correct namespace.

Maybe you can identify this issue. What we are doing wrong? or What is the correct approach?

## Occurs in next-i18next version
```
"next": "^9.5.4",
"next-i18next": "^4.5.0",
```

## Steps to reproduce
I created a [minimal repo](https://github.com/ahmadHuss/hello) you can check the implementation and if you have time you can also run this in your local environment. The sample is also [live](https://hello-new-i18.herokuapp.com/) for the test. Currently, inside the [minimal repo](https://github.com/ahmadHuss/hello) 5 locales are created i.e.
```
en => English i.e. default
pt => Português
sv => Svenska
sk => Slovenčina
tl => Tagalog
```

## Expected behaviour
When a page or pages with different locales are loaded, It should load with the correct namespace.

## Screenshots
![sample-problem](https://user-images.githubusercontent.com/15076149/100923458-03355b80-3501-11eb-9336-af154f34426f.gif)

## OS (please complete the following information)
Device: Windows 10
Browser: Chrome, ios safari ... etc

## Additional context
Your work and contributions to the open-source are great.

**Comment by isaachinman** [comment]
@ahmadHuss Sorry, that's quite an old version of `next-i18next`, and it looks like you're still using an express server/middleware. Unless you were willing to upgrade to the latest version of the package, I'm not sure I would be able to help debug.

**Comment by ahmadHuss** [comment]
@isaachinman Thanks for the reply. I upgraded the dependencies with the latest `next-118nextv7.0.1` and `nextjsv9.5.4`. You can check the [minimal repo code](https://github.com/ahmadHuss/hello) and [live sample link](https://hello-new-i18.herokuapp.com/).  I tried to check it again but the problem still happening. i.e. sometimes on the initial page load namespace will not correct load but after the refresh, it will load a correct namespace.

![problem](https://user-images.githubusercontent.com/15076149/101126707-d9258b80-361d-11eb-853a-1e7f2cfe245e.gif)

**Comment by isaachinman** [comment]
Hi @ahmadHuss – I have cloned your repo, but cannot reproduce the issue you have demonstrated in your gif either locally or on your Heroku deploy. Do you have any further information on how to reproduce?

**Comment by ahmadHuss** [comment]
@isaachinman You mean when you open links in a tab, all the namespaces are loading correctly. Sometimes the namespaces are loading fine but when you open these links **immediately** on a new tab holding `Command key + click` on `MAC` You will find the namespaces for some links are not correctly loaded.

If you still no reproduce this issue. You can comment.

**Comment by isaachinman** [comment]
Yes, I have opened the tabs exactly as you've done in your gif, and I cannot get a reproduction.

**Comment by ahmadHuss** [comment]
@isaachinman Can you please upload a gif so I can see why it doesn't behave like a GIF?

**Comment by ahmadHuss** [comment]
@isaachinman Can you please upload a gif so I can understand why it doesn't behave like a GIF?

**Comment by isaachinman** [comment]
Hi there, I have just released `next-i18next@8.0.0`, which uses the internationalised routing built into NextJs v10+. This new version of `next-i18next` is quite different, thus making this PR no longer relevant. Thanks!
